source/_layouts/page.html

```diff
@@ -18,8 +25,10 @@ <h1 class="text-black text-[24px] md:text-[44px] font-extrabold tracking-[-0.02e
                 {{ page.title }}
             </h1>
             <div class="content post-content">
-                {{ page.content | markdownify | output_modder }}
+                {{ content }}
```

Comment:
Hmm, this removes the output handling we have for handling things like automatically adding `rel='external nofollow'` to outgoing links.

https://github.com/home-assistant/home-assistant.io/blob/982dfd1e9e43f072b79900e9d8d4f1adc40b6371/plugins/output_modder.rb#L6

Is there a specific reason this was removed?

Reply:
The reason this was removed was that some elements such as the includes were not displayed correctly:
![image](https://github.com/home-assistant/home-assistant.io/assets/38767475/6def7d40-5450-4ba8-880e-9953cf995f4f)

But it turns out that this was caused by `page.content`.

Reply:
👍 

Nice!